bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/util/AbstractBaseImpl.java

```diff
@@ -61,6 +62,10 @@ public abstract class AbstractBaseImpl extends AbstractFormComponentImpl impleme
     @Named("fd:formtype")
     protected String dorTemplateType;
 
+    @Nullable
+    @ChildResource(name = "fd:signerInfo", injectionStrategy = InjectionStrategy.OPTIONAL)
```

Comment:
Please reuse the constant name. Why is this in baseImpl ? Signer info would only be present in form container